lib/server.go

```diff
@@ -709,9 +718,14 @@ func NewServer(
 		if stateChangeSyncer != nil {
 			stateChangeSyncer.Reset()
 		}
-		glog.Errorf(CLog(Red, "NewServer: Forcing a rollback to the last snapshot epoch because node was not closed "+
-			"properly last time"))
-		return nil, errors.Wrapf(err, "NewServer: Restart required"), true
+		if !_forceChecksum && isChecksumIssue {
```

Comment:
@lazynina I think you have this backwards. We want to return an error in the first part of the if, not the second. Would swap the two.